core/src/main/resources/swagger/rawls.yaml

```diff
@@ -596,6 +596,37 @@ paths:
              - openid
              - email
              - profile
+  '/api/admin/project/registration':
+    post:
+      tags:
+        - admin
+      operationId: recordProjectOwnership
+      summary: create a record of project ownership
+      parameters:
+        - name: project
+          description: billing project ID
+          required: true
+          in: query
+          type: string
+        - name: bucket
+          description: Cromwell auth bucket
+          required: true
+          in: query
+          type: string
+      responses:
+        201:
+          description: Successful project record creation
+        403:
+          description: You must be an admin to call this endpoint.
+        404:
+          description: Missing required parameter
+        500:
+          description: Internal Server Error
+      security:
+        - authorization:
+          - openid
+          - email
+          - profile
```

Comment:
missing deregister swagger

Reply:
wait. there is no deregister? i forgot -- what are we doing to erase sam policies then?

Reply:
oh, tracking at the all-the-tests level. bah, i forgot this

Reply:
Correct. Seeing whether the clean-up of Sam policies can be done at the test afterAll level is probably one of the first things to look at next.